pages/pixel-ui/avatar/set-steamid.mdx

```diff
@@ -0,0 +1,17 @@
+import Method from "/components/method";
+import Argument from "/components/method/argument";
+import ReturnValue from "/components/method/return_value";
+
+# SetSteamID
+<Method
+	name="Avatar:SetSteamID"
+	realm="shared-menu"
+	args={[
+		<Argument type="string" name="id">The 64bit SteamID of the player to load avatar of</Argument>,
+		<Argument type="number" name="size">The size of the avatar to use. Acceptable sizes are 32, 64, 184.</Argument>
+	]}
+	repositoryUrl="https://github.com/TomDotBat/pixel-ui"
+	declaration="lua/pixelui/elements/cl_avatar.lua#L47"

```

Comment:
Usage of multi-line selections is preferred for adding method declaration links, this one should be `#L47-L49`.